DataLayer/CmuData/FakeCmuUnit.cpp

```diff
@@ -0,0 +1,138 @@
+#include "FakeCmuUnit.h"
+
+FakeCmuUnit::FakeCmuUnit()
+{
+}
+
+FakeCmuUnit::~FakeCmuUnit()
+{
+}
+
+/* CmuUnit Gets */
+unsigned char FakeCmuUnit::getCmuNumber() const
+{
+    return 1;
+}
+unsigned short FakeCmuUnit::getCell0Voltage() const
+{
+    return 2;
+}
+unsigned short FakeCmuUnit::getCell1Voltage() const
+{
+    return 3;
+}
+unsigned short FakeCmuUnit::getCell2Voltage() const
+{
+    return 4;
+}
+unsigned short FakeCmuUnit::getCell3Voltage() const
+{
+    return 5;
+}
+unsigned short FakeCmuUnit::getCell4Voltage() const
+{
+    return 6;
+}
+unsigned short FakeCmuUnit::getCell5Voltage() const
+{
+    return 7;
+}
+unsigned short FakeCmuUnit::getCell6Voltage() const
+{
+    return 8;
+}
+unsigned short FakeCmuUnit::getCell7Voltage() const
+{
+    return 9;
+}
+unsigned short FakeCmuUnit::getPcbTemp() const
+{
+    return 10;
+}
+unsigned short FakeCmuUnit::getCellTemp0() const
+{
+    return 11;
+}
+unsigned short FakeCmuUnit::getCellTemp1() const
+{
+    return 12;
+}
+unsigned short FakeCmuUnit::getCellTemp2() const
+{
+    return 13;
+}
+unsigned short FakeCmuUnit::getCellTemp3() const
+{
+    return 14;
+}
+unsigned short FakeCmuUnit::getCellTemp4() const
+{
+    return 15;
+}
+unsigned short FakeCmuUnit::getCellTemp5() const
+{
+    return 16;
+}
+unsigned short FakeCmuUnit::getCellTemp6() const
+{
+    return 17;
+}
+unsigned short FakeCmuUnit::getCellTemp7() const
+{
+    return 18;
+}
+unsigned short FakeCmuUnit::getCellTemp8() const
+{
+    return 19;
+}
+unsigned short FakeCmuUnit::getCellTemp9() const
+{
+    return 20;
+}
+unsigned short FakeCmuUnit::getCellTemp10() const
+{
+    return 21;
+}
+unsigned short FakeCmuUnit::getCellTemp11() const
+{
+    return 22;
+}
+unsigned short FakeCmuUnit::getCellTemp12() const
+{
+    return 23;
+}
+unsigned short FakeCmuUnit::getCellTemp13() const
+{
+    return 24;
+}
+unsigned short FakeCmuUnit::getCellTemp14() const
+{
+    return 25;
+}
+
+/* CmuUnit Sets */
+void FakeCmuUnit::setCmuNumber(const unsigned char& cmuNumber) {}
+void FakeCmuUnit::setCell0Voltage(const unsigned short& cell0Voltage) {}
+void FakeCmuUnit::setCell1Voltage(const unsigned short& cell1Voltage) {}
+void FakeCmuUnit::setCell2Voltage(const unsigned short& cell2Voltage) {}
+void FakeCmuUnit::setCell3Voltage(const unsigned short& cell3Voltage) {}
+void FakeCmuUnit::setCell4Voltage(const unsigned short& cell4Voltage) {}
+void FakeCmuUnit::setCell5Voltage(const unsigned short& cell5Voltage) {}
+void FakeCmuUnit::setCell6Voltage(const unsigned short& cell6Voltage) {}
+void FakeCmuUnit::setCell7Voltage(const unsigned short& cell7Voltage) {}
```

Comment:
Can these functions be condensed to just `setCellVoltage` and take 2 arguments, the voltage and the cell number? This goes for the temps below as well.